ext/TransitionVisualizations.jl

```diff
@@ -0,0 +1,229 @@
+module TransitionVisualizations
+
+using TransitionsInTimeseries, Makie
+
+# Default options for plotting utilities
+const default_accent_linewidth = 3
+const default_colors = ["#7143E0", "#0A9A84", "#191E44", "#AF9327", "#701B80", "#2E6137"]
+
+default_indicator_label(res::IndicatorsChangesResults) = [string(nameof(
+    res.config.indicators[i])) for i in eachindex(res.config.indicators)]
+
+function default_chametric_label(res::IndicatorsChangesResults)
+    labels = String[]
+    for i in eachindex(res.config.change_metrics)
+        if res.config.change_metrics[i] isa PrecomputedRidgeRegressionSlope ||
+            res.config.change_metrics[i] isa RidgeRegressionSlope
+            push!(labels, "Regression slope")
+        else
+            push!(labels, string(nameof(res.config.change_metrics[i])))
+        end
+    end
+    return labels
+end
+
+"""
+    TransitionVisualization
+Contains:
+ - `fig::Figure` the figure,
+ - `laxs::Vector{Axis}` the left axes,
+ - `raxs::Vector{Axis}` the right axes,
+ - `res::IndicatorsChangesResults` the results of the input signal,
+ - `colors::Vector` a vector containing the colors to use for plotting.
+"""
+struct TransitionVisualization{R<:IndicatorsChangesResults}
+    fig::Makie.Figure
+    laxs::Vector{Makie.Axis}
+    raxs::Vector{Makie.Axis}
+    res::R
+    colors::Vector
+end
```

Comment:
This seems convoluted. Why do we need to store all this information? Why can't `plot_changes_significance` just return the `Figure` object? Note that you can access axes from the figure. `content(fig[1,1])` gives the axis in location [1,1]. I think we can do away from even needing this `TransitionVisualization` thing, and `plot_significance!` can act directly on the Figure object.

Reply:
I agree. `content(fig[1,1])` however does not work here, because we have 2 axes per subfigure. To tackle this, I introduced a matrix layout for the axes. Plot on the left axis by calling first column and on the right axis by calling the second. For instance:

```julia
lines!(axs[i, 1], t_cha, x_cha)
lines!(axs[i, 2], t_ind, x_ind)
```

Thus, we got rid of the struct `TransitionVisualization`.